lib/senders/sms.js

```diff
@@ -4,72 +4,72 @@
 
 'use strict'
 
-var Nexmo = require('nexmo')
-var MockNexmo = require('../mock-nexmo')
+var AWS = require('aws-sdk')
 var P = require('bluebird')
 var error = require('../error')
 
 module.exports = function (log, translator, templates, config) {
   var smsConfig = config.sms
-  var nexmo = smsConfig.useMock ? new MockNexmo(log, config) : new Nexmo({
-    apiKey: smsConfig.apiKey,
-    apiSecret: smsConfig.apiSecret
+  var SNS = new AWS.SNS({
+    region: smsConfig.apiRegion
   })
 
-  var sendSms = promisify('sendSms', nexmo.message)
-  var NEXMO_ERRORS = new Map([
-    [ '1', error.tooManyRequests(smsConfig.throttleWaitTime) ]
-  ])
-
   return {
-    send: function (phoneNumber, senderId, templateName, acceptLanguage, signinCode) {
+    send: function (phoneNumber, templateName, acceptLanguage, signinCode) {
       log.trace({
         op: 'sms.send',
-        senderId: senderId,
         templateName: templateName,
         acceptLanguage: acceptLanguage
       })
 
       return P.resolve()
         .then(function () {
           var message = getMessage(templateName, acceptLanguage, signinCode)
+          var params = {
+            Message: message.trim(),
+            MessageAttributes: {
+              'AWS.SNS.SMS.MaxPrice': {
+                // The maximum amount in USD that you are willing to spend to send the SMS message.
+                DataType: 'String',
+                StringValue: '1.0'
+              },
+              'AWS.SNS.SMS.SenderID': {
+                // Up to 11 alphanumeric characters, including at least one letter and no spaces
+                DataType: 'String',
+                StringValue: 'Firefox'
```

Comment:
May be worth keeping the sender id in config, even if it is the same for all regions. It's unlikely but I could imagine, for instance, a request to change it for product/marketing reasons, which we may not want to cut a release for.